### DISCUSSION TITLE
can not run in win 10

### DISCUSSION THREAD

**Discussion by mathrt** [unageek/graphest#626] [Q&A] [answered]
Graphest is a so wonderful  mathematical function graph software.  After installing the .exe file in window 10 21H1, I can not open and use it. Nothing happens when I double click. How can I solve this problem? Thank you very much.

**Comment by unageek** [comment]
Hi! I have only tested with Windows 11 and not been able to reproduce the issue. Do you have relevant logs in the Event Viewer?

**Comment by unageek** [comment, marked as answer]
If this was due to #757, it will be fixed in the next release.
